README.md

```diff
@@ -25,27 +27,62 @@ or install from source:
 ```
 pip install git+https://github.com/onnx/onnxmltools
 ```
-Note:If you choose to install `onnxmltools` from its source code, you must set an environment variable `ONNX_ML=1` before installing `onnx` package.
+If you choose to install `onnxmltools` from its source code, you must set an environment variable `ONNX_ML=1` before installing `onnx` package.
```

Comment:
can we make ONNX_ML=1 being a default behavior?

Reply:
Unfortunately, no. We have some partners focusing on NN only.